diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -44,6 +44,11 @@ function StudentQuiz(runtime, element, initData) {
       currentAnswerContainer = ".diagnostic-feedback .current",
       studentViewFormSecSelector = ".diagnostic-feedback .student_view_form section",
       questionId = '.question-id',
+      lessonContentSelector = '.lesson-content',
+      contentSelector = ".content",
+      questionContaner = '.q-container',
+      quizQuestion = '.quiz-question',
+      userAnswers = '.user-answers',
```

Comment:
@jareerahsan Thanks for making the code in `getQuestionEventData` rely more on CSS classes than relative positioning for finding elements.

A couple notes about these selectors:

* It would be good to make the naming scheme consistent. Currently, some of the selectors have `Selector` in their name, and others don't. My preference would be to remove the `Selector` part from the variable names: I had a look at some of the `*Selector` variables in context, and since they're only ever used as a first argument for `$` (which is great), the `Selector` part is a bit noisy. (In other words, the context in which you're using the variables makes it clear that they are selectors; there's no need to reiterate that by adding `Selector` to their names.)
* Typo: `questionContaner` ⟶ `questionContainer`
* What is the difference between `.lesson-content` and `.content`? Might be worth rethinking the names of these CSS classes (and adjusting the variable names accordingly).